USAGE.md

```diff
@@ -231,10 +231,14 @@ After enforcing the boundary checks for a package, you may execute:
 
 Packwerk will check the entire codebase for any new or stale violations.
 
-You can also specify folders for a shorter run time:
+You can also specify folders for a shorter run time. When checking against folders all subfolders will be analyzed, irrespective of nested package boundaries.
 
     packwerk check components/your_package
 
+You can also specify packages for a shorter run time. When checking against packages all subfolders will be analyzed and nested packaged boundaries will be honored.
```

Comment:
I would be more explicit and say that if you specify a _package_ to check, packages nested inside of it will not be checked.

Reply:
I don't think it's immediately apparent what "honoring nested package boundaries" means in this context.

Reply:
Makes sense - I will push a fix